src/panels/Scene/SceneGraphNode/SceneGraphNodeHeader.tsx

```diff
@@ -51,12 +55,28 @@ export function SceneGraphNodeHeader({ uri, onClick, label }: Props) {
     </Button>
   );
 
+  // TODO: Make sure that the timeframe information is accessible
+  const visibilityControl = hasRenderable && (
+    <Group gap={'xs'}>
+      <PropertyOwnerVisibilityCheckbox uri={renderableUri} />
+      {timeFrame && !isInTimeFrame && (
+        <Tooltip
+          label={`This node is currently hidden since the time is outside its specified
+            time frame. It will not be visible even if enabled.`}
```

Comment:
```suggestion
          label={`This node is hidden because the current time is outside its specified
            time frame. It will not be visible even if enabled.`}
```
?